analyticsdataserver/settings/base.py

```diff
@@ -293,6 +302,7 @@
 ########## ANALYTICS DATA API CONFIGURATION
 
 ANALYTICS_DATABASE = 'default'
+APPS_LABELS_FOR_ANALYTICS_DATABASE = ('v0', 'enterprise_data')
```

Comment:
What's going on here? The `app_label` for the existing API is `v0`? That's a bummer.

I wouldn't make this a setting. Just make a list [here](https://github.com/edx/edx-analytics-data-api/pull/184/files#diff-110b488f2b07b14fdb4a2589e7da9e18R10).